### PR TITLE
[TextFields] Add FullWidth snapshot tests.

### PR DESCRIPTION
**Pull request by romoore**
Adding basic snapshot tests for full-width Text Fields.

Part of #5762

### CODE REVIEW

File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerSnapshotTests.m [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+  [self triggerTextFieldLayout];
+  UIView *snapshotView = [self addBackgroundViewToView:self.textField];
+
+  // Perform the actual verification.
+  [self snapshotVerifyView:snapshotView];
+}
+
+#pragma mark - Tests
+
+- (void)testOutlinedTextFieldEmpty {
```
Comment (jverkoey) [inline components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerSnapshotTests.m]: Do the names need to be updated? It looks like these aren’t outlined text fields.

**Review by jverkoey — APPROVED** [review]

File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerSnapshotTests.m (hunk 1 above)
Comment (rami-a) [reply components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerSnapshotTests.m]: Yeah I believe a find/replace is needed

**Review by rami-a — APPROVED** [review]

File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerSnapshotTests.m (hunk 1 above)
Comment (romoore) [reply components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerSnapshotTests.m]: Ah yeah. I wonder if I should make the method names more generic so the copy-paste will be easier...

Comment (romoore) [reply components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerSnapshotTests.m]: ( -_-) Oops.... Done

**Pull request merged**